src/warc2zim/replay/topFrame.html

```diff
@@ -29,10 +34,31 @@
   // update URL when iframe changes
   window.addEventListener("message", function() {
     if (event.data.wb_type === "load" || event.data.wb_type === "replace-url") {
+
+      if (event.data.title) {
+        document.title = event.data.title;
+      }
       
       // remove scheme to be consistent with current canonicalization
       const urlNoScheme = event.data.url.slice(event.data.url.indexOf("//") + 2);
       window.history.replaceState(null, "", PREFIX + urlNoScheme);
+
+      // if icons received, replace any existing icons with new ones
+      if (event.data.icons) {
```

Comment:
does hat mean that browsing twitter.com will bring twitter favicon but browsing to somthing.net (with no favicon) will keep twitter favicon?

Reply:
good question! I don't remember that happening in other environment, but will double check!

Reply:
To confirm, yes, if there is no icon, this will be an empty list, and `if([])` if true in JS, so it will just remove any existing icons